### PR TITLE
Delete Product

### PR DESCRIPTION
**Pull request by liushiru**

### CODE REVIEW

**Comment by liushiru** [comment]
Please add exception handling before the next milestone

File: src/main/java/duke/logic/command/order/DeleteOrderCommand.java [hunk 1]
```diff
@@ -24,7 +24,7 @@
     private final Set<Index> indices;
 
     /**
-     * Creates a {@code DeleteOrderCommand}.
+     * Creates a {@code DeleteProductCommand}.
```
Comment (liujiajun) [inline src/main/java/duke/logic/command/order/DeleteOrderCommand.java]: Why the change?

Comment (liushiru) [reply src/main/java/duke/logic/command/order/DeleteOrderCommand.java]: Sorry I think I accidentally changed it...
